airbyte-integrations/connectors/source-microsoft-teams/source_microsoft_teams/client.py

```diff
@@ -102,8 +106,6 @@ def _make_request(self, api_url: str, params: Optional[Dict] = None) -> Union[Di
 
     @staticmethod
     def _get_response_value_unsafe(raw_response: Dict) -> List:
-        if "value" not in raw_response:
```

Comment:
why was this removed? your description in the PR made it sound like this field might not always be populated.